code/datums/status_effects/buffs/food/chilling.dm

```diff
@@ -0,0 +1,13 @@
+///food effect applied by ice cream and frozen treats
+/datum/status_effect/food/chilling
+	alert_type = /atom/movable/screen/alert/status_effect/icecream_chilling //different path, so we sprite one state and not five.
+
+/datum/status_effect/food/chilling/tick(seconds_between_ticks)
+	var/minimum_temp = (BODYTEMP_HEAT_DAMAGE_LIMIT - 12 * strength)
```

Comment:
We have BODYTEMP_HEAT_DAMAGE_LIMIT_ONE/TWO/THREE, not really much importance as the temperature change is so low when compared to being set on fire and such, but i don't think we have this defined without the specification of one, two, or three.

Reply:
I have no idea what you reviewed since I haven't gotten there yet but I believe you, I have like 10 files touched out of the 137 ported so far

Reply:
Was just looking at things that concern me directly, such as things that would aid or hinder my gameplay as a Vali man. The minor speed buff from some food is very interesting, as well as Reagent Effect Multiplier(REM) workaround thing as TG changed from a 0.4 food metabolism rate to whatever they have now. Threw me for a loop, but it'll probably only be of minor importance.

Reply:
In before icecream vali meta